Cargo.toml

```diff
@@ -11,17 +11,14 @@ repository = "https://github.com/init4tech/zenith"
 license = "AGPL-3.0"
 
 [dependencies]
+alloy = { version = "0.9", features = ["full", "json-rpc", "signer-aws", "rpc-types-mev"] }
```

Comment:
Using this opportunity to bump alloy to 0.9 unless anyone has issues with that.

Reply:
if this doesn't affect any downstream dependents, that's cool!

Reply:
Turns out it can't be bumped, it will cause other errors. Reverting it back to what it was before 🚒